api/accounts/accounts.go

```diff
@@ -354,21 +354,27 @@ func (a *Accounts) Mount(root *mux.Router, pathPrefix string) {
 
 	sub.Path("/*").
 		Methods(http.MethodPost).
-		HandlerFunc(utils.MetricsWrapHandlerFunc(pathPrefix, "accounts_call_batch_code", a.handleCallBatchCode))
+		Name("accounts_call_batch_code").
+		HandlerFunc(utils.WrapHandlerFunc(a.handleCallBatchCode))
 	sub.Path("/{address}").
 		Methods(http.MethodGet).
-		HandlerFunc(utils.MetricsWrapHandlerFunc(pathPrefix, "accounts_get_account", a.handleGetAccount))
+		Name("accounts_get_detail").
```

Comment:
Would be good to keep the name as close as possible to the URL and avoid the interpretation step.
Ie. `curl .../accounts/0x0000` -> `accounts_get_account`